### PR TITLE
3.5: re-add the ability to emmit errors from arangodb user defined functions

### PR DESCRIPTION
**Pull request by dothebart**

### CODE REVIEW

**Comment by dothebart** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/10688/

**Comment by dothebart** [comment]
forward port of https://github.com/arangodb/arangodb/pull/11995

File: CHANGELOG [hunk 1]
```diff
@@ -1,6 +1,8 @@
 v3.5.6 (XXXX-XX-XX)
 -------------------
 
+* re-add AQL_WARNING() to emmit errors from UDFs
```
Comment (Simran-B) [inline CHANGELOG:4]:
```suggestion
* re-add AQL_WARNING() to emit warnings from UDFs
```

**Comment by dothebart** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/10691/

**Comment by dothebart** [comment]
all blue.

**Pull request merged**